### ISSUE TITLE
Question: business rules

### ISSUE THREAD

**Issue by alexKatona** [ehn-dcc-development/hcert-kotlin#52]
Hello,

saw that you @nodh  added some code to store the business rules as Json, is there gonna be an interoperability between this library and https://github.com/ehn-dcc-development/dgc-business-rules ? Since hcert-kotlin already contains data and valueSets, it could technically generate data structure that can be used with the business rules.

**Comment by nodh** [comment]
You're right, we could add a data class to actually hold a business rule, i.e. one that deserializes the JSON strings we're storing. Unfortunately, this is not on our roadmap in the near future.
